common/thorhelper/thorsoapcall.cpp

```diff
@@ -2559,10 +2559,16 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
             {
                 checkTimeLimitExceeded(&remainingMS);
                 checkRoxieAbortMonitor(master->roxieAbortMonitor);
-                OwnedSpanScope socketOperationSpan = master->activitySpanScope->createClientSpan("Socket Write");
-                setSpanURLAttributes(socketOperationSpan, url);
 
-                Owned<IProperties> traceHeaders = ::getClientHeaders(socketOperationSpan);
+                StringBuffer spanName;
+                spanName.appendf("%s %s %s:%d", getWsCallTypeName(master->wscType), master->service.str(), url.host.str(), url.port);
```

Comment:
Have you discussed this change with any of the potential users?  Does it fit in with their conventions?

I would not expect the name to include the url or the port - since you ideally want to be able to aggregate over all instances.

Reply:
The proposed format change came about from a conversation between us. I've added a comment to the original jira asking for feedback, however I'm certain there are no current consumers, and no expected format. 

https://hpccsystems.atlassian.net/browse/HPCC-31971?focusedCommentId=61883

Reply:
@ghalliday we've received at least one positive comment on proposed change in Jira.